docs/en/05_Contributing/02_Build_Tooling.md

```diff
@@ -78,6 +81,13 @@ The first `--` separator is required to separate arguments from NPM's own ones.
 $ npm run build -- --watch
 ```
 
+*For development only*:
+Run this to keep webpack automatically rebuilding your file changes, this will also include *.map files for easier debugging.
+It is important to note that this should not be used for pushing up changes, and you should run `npm run build` after you're done.
+```
+$ npm run watch
```

Comment:
We should use `yarn run` rather than `npm run` consistently here. I'm assuming those have the same effect